.github/README.md

```diff
@@ -39,6 +40,81 @@ cd  # in your home directory
 git clone https://github.com/torvalds/linux.git
 ```
 
+If you are compiling the kernel for the first time, install some of the packages required:
+
+On a Debian/Ubuntu distribution:
+
+
+```
+sudo apt install -y \
+  qtbase5-dev \
+  pkg-config \
+  bison \
+  flex \
+  libelf-dev \
+  libssl-dev \
+  libcap-dev \
+  llvm
+```
+
+Enable the folliwing config options to ensure for `hawk` to work:
+
+```
+make defconfig && make kvm_guest.config && \
+  scripts/config \
+  -e BPF_SYSCALL \
+  -e BPF_LSM \
+  -e BPF_JIT \
+  -e DEBUG_INFO \
+  -e DEBUG_INFO_BTF \
+  -e FTRACE \
+  -e DYNAMIC_FTRACE \
+  -e FUNCTION_TRACER && \
+  make olddefconfig
+```
+
+If you are doing kernel debugging, you may additionally want to enable:
+
+```
+scripts/config \
+  -e DEBUG_ATOMIC_SLEEP \
+  -e KASAN \
+  -e KMEMLEAK \
+  -e PROVE_LOCKING \
+  -e SECURITYFS \
+  -e CGROUP_BPF \
+  -e IKCONFIG_PROC &&
+  make olddefconfig
+```
+
+It's also recommended to build and install the latest `clang` and `pahole`, especially if
+you are working on the latest Linux kernel:
+
+**clang**
+
+```
+sudo apt install -y cmake && \
+git clone https://github.com/llvm/llvm-project.git  && cd llvm-project && \
+mkdir build && \
+cd build && \
+cmake -DLLVM_ENABLE_PROJECTS=clang -DCMAKE_BUILD_TYPE=Release -G "Unix Makefiles" ../llvm && \
+make -j $(nproc) && \
```

Comment:
apparently `make -j` is what the cool kids do these days (same for pahole)

Reply:
Thanks! Done.